cfgov/unprocessed/js/modules/VideoPlayer.js

```diff
@@ -0,0 +1,206 @@
+/* ==========================================================================
+    Base Video Player Class
+   ========================================================================== */
+
+'use strict';
+
+var _assign = require( './util/assign' ).assign;
+var _noopFunct = require( './util/standard-type' ).noopFunct;
+var DOM_INVALID = require( '../config/error-messages-config' ).DOM.INVALID;
+var jsLoader = require( './util/js-loader' );
+
+var CLASSES = Object.freeze( {
+    VIDEO_PLAYER_SELECTOR:     '.video-player',
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/pull/1625/commits/36f6195becc4c28009ba7b84c31290ffe07277ae.